packages/terra-modal/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "terra-modal",
+  "main": "lib/Modal.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Modals are used to display content in a layer above the app. Modals consist of an overlay and container. Higher order components can provide the modal dialog with a header, body, and close button.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-modal",
+    "Modal",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-icon": "0.x",
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "focus-trap-react": "^3.0.2",
+    "terra-mixins": "^1.0.0",
+    "react-portal": "^3.0.0"
```

Comment:
Does this component need an explicit terra-base dependency? 
https://github.com/cerner/terra-core/pull/251